plugins/wordcount/plugin.js

```diff
@@ -188,6 +191,38 @@ CKEDITOR.plugins.add("wordcount", {
             return html;
         }
 
+        function doSEOCheck(text, editorInstance) {
+            text = getTextBeforeFirstSEOObject(text);
+            let wordCount = countWords(text);
+            toggleSEO(editorInstance, wordCount);
+        }
+
+        function getTextBeforeFirstSEOObject(text) {
+            var assetCardSeperator = '<div asset="::asset"';
+            var relatedLinkSeperator = '<span class="exclude-from-newsgate';
+            var indexOfAssetCard = text.indexOf(assetCardSeperator);
```

Comment:
Why car and not let?

Reply:
To keep it consistent with the plugin file. Should we use the forked version in some project other than Presto. Again more of consistency than anything else.